js/main.js

```diff
@@ -69,18 +67,42 @@ var $newEntry = document.querySelector('div[data-view="entry-form"]');
 var $entriesLink = document.querySelector('a[href="#entry-form"]');
 
 $newButton.addEventListener('click', function () {
-  $newEntry.className = '';
-  $entriesPage.className = 'hidden';
-  data.view = "entry-form"
+  switchViews($newEntry.getAttribute('data-view'));
+  data.view = 'entry-form';
```

Comment:
You could assign a value to `data.view` inside the `switchViews` function so that you don't need to remember to do it every time you switch views 🤓